rudder-agent/SOURCES/uuidgen

```diff
@@ -1,6 +1,7 @@
 #!/bin/sh
 
 # uuidgen doesn't exist on AIX, so provide a compatible shell function
+# This is not compatible with Linux systems.
```

Comment:
@jooooooon Because I wasn't sure how to do it, and even if we should really do it this way, see my comment below.